modules/gateways/Blockonomics/Blockonomics.php

```diff
@@ -234,14 +278,22 @@ public function getMargin() {
 			->where('setting', 'Margin')
 			->value('value');
 	}
+
 	/*
 	 * Convert fiat amount to BTC
 	 */
-	public function getBitcoinAmount($fiat_amount, $currency) {
+	public function getNewAmount($fiat_amount , $blockonomics_currency = 'btc') {
```

Comment:
Please choose a better function name. It is not very clear what this function is doing

Reply:
Changed in d3329f4